brainiak/fcma/preprocessing.py

```diff
@@ -127,9 +203,11 @@ def prepare_fcma_data(images, conditions, mask1, mask2=None,
             activity_data1, activity_data2 = zip(*multimask_images(images,
                                                                    masks,
                                                                    np.float32))
+            _randomize_subject_list(activity_data2, random)
             raw_data2, _ = _separate_epochs(activity_data2, conditions)
         else:
             activity_data1 = list(mask_images(images, mask1, np.float32))
+        _randomize_subject_list(activity_data1, random)
```

Comment:
Thanks for the suggestion! Somehow I ignored the return value here although I should put it as `activity_data1 = _randomize_subject_list(activity_data1, random)`. I would prefer working in-place, let me modify `_randomize_subject_list` accordingly.